### PR TITLE
Feature/python conda upgrade

### PR DESCRIPTION
**Pull request by SiggyF**
- Use mamba for improved installation performance
- apt-get clean to reduce image size
- install packages available in anaconda through anaconda

### CODE REVIEW

**Review by Mares2022 — APPROVED** [review]

File: Dockerfile
Comment (Mares2022) [inline Dockerfile:1]: Remove commented lines 9-10.
It is unclear what is ARG MAMBA_DOCKERFILE_ACTIVATE=1 used for.

File: env.yaml
Comment (Mares2022) [inline env.yaml:1]: Clear changes.

File: requirements.txt
Comment (Mares2022) [inline requirements.txt:1]: Remove commented or unused packages.

File: setup.py
Comment (Mares2022) [inline setup.py:1]: Clear changes.

File: Dockerfile
Comment (SiggyF) [reply Dockerfile:1]: Documented

File: requirements.txt
Comment (SiggyF) [reply requirements.txt:1]: Removed

**Pull request merged**